bolts/data/datasets/audio/__init__.py

```diff
@@ -1 +1 @@
-from .ecoacoustics import *
+from .ecoacoustics_ds import *
```

Comment:
the filename can drop `_ds` too

Reply:
Good catch!

Reply:
^^ All changed and tested, works renamed.